ingest/pom.xml

```diff
@@ -88,8 +88,6 @@
                     <ignoredUnusedDeclaredDependencies>
                         <dependency>org.xerial:sqlite-jdbc</dependency>
                         <dependency>org.slf4j:slf4j-simple</dependency>
-                        <dependency>org.junit.jupiter:junit-jupiter-engine</dependency>
```

Comment:
Why do we need this? The first is already scoped for tests only, and if the second is just for test scope why not scope for tests only?